connexion/decorators/security.py

```diff
@@ -252,7 +252,7 @@ def verify_apikey(apikey_info_func, loc, name):
 
     def wrapper(request, required_scopes):
         if loc == 'query':
-            apikey = request.query.get(name)
+            apikey, request.query = _immutable_pop(request.query, name)
```

Comment:
I think you'll also need to handle the `KeyError` if `name` is not in `request.query`.

Reply:
Yes, indeed. I've added try / except here.